OHHTTPStubs.podspec

```diff
@@ -1,7 +1,7 @@
 Pod::Spec.new do |s|
 
   s.name         = "OHHTTPStubs"
-  s.version      = "6.1.0"
+  s.version      = "6.1.1"
```

Comment:
You can revert this change, my scripts will take care of updating the podspec with the future version number when we decide to release a new version and determine what that version number should be 😉

Reply:
👌